dotnetv3/SQS/SQSReceiveMessagesExample/SQSReceiveMessagesExample/SQSReceiveMessagesExample.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+/// <summary>
+/// Shows how to receive messages from an existing Amazon SQS queue.

```

Comment:
First mention should be: "Amazon Simple Queue Service (Amazon SQS)" and subsequent mentions: "Amazon SQS"